guidelines/1.3.2.md

```diff
@@ -1,46 +1,94 @@
+[Help us by reporting issues via Github](https://github.com/theappbusiness/accessibility-guidelines) or [email](mailto:jeanfrancois@theappbusiness.com)
+
+[Back to the overview page](./../README.md)
```

Comment:
I'm torn about this. Isn't the back button in people's browsers enough for this?

Reply:
I'm imagining that people will bookmark/share links to the detail pages directly. Then, I want them to have an easy way to find the overview page again.

Reply:
It could be that re replace that 'back link' with a [breadcrumbs pattern](https://design-system.service.gov.uk/components/breadcrumbs/) when we move to HTML.